src/ServiceControl/SagaAudit/ListSagasApi.cs

```diff
@@ -0,0 +1,32 @@
+namespace ServiceControl.SagaAudit
+{
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Nancy;
+    using Raven.Client;
+    using ServiceControl.CompositeViews.Messages;
+    using ServiceControl.Infrastructure.Extensions;
+
+    public class ListSagasApi : ScatterGatherApi<NoInput, SagaListIndex.Result[]>
```

Comment:
Would it make sense to use here IList because that is was is returned from raven but then does newtonsoft support IList?